src/main/java/com/shipmentEvents/handlers/EventHandler.java

```diff
@@ -198,11 +198,23 @@ public ModelAndView inputSanitizationNonCompliant(@RequestParam String favoriteC
     }
     
     public void processResponseNoncompliant(Map<String, String> response) {
-    // Noncompliant: logs sensitive data without redaction resulting in a possible sensitive information leak.
-    final String name = response.get("Name");
-    String redacted = RedactionUtil.redact(name);
-    log.info(name);
-}
+        // Noncompliant: logs sensitive data without redaction resulting in a possible sensitive information leak.
+        final String name = response.get("Name");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This code contains a potential information leakage in the handling of the following *logged and redcated data*: (String name). 

**Fix**
Release data more selectively, or declassify the data before it&#39;s released. 

**More info**
[Learn more about information leakage](https://cwe.mitre.org/data/definitions/532.html) (external link).